sopel/bot.py

```diff
@@ -417,6 +421,13 @@ def reply(self, message, destination=None, reply_to=None, notice=False):
                 reply_to = self._trigger.nick
             self._bot.reply(message, destination, reply_to, notice)
 
+        def kick(self, nick, channel=None, message=''):
+            if channel is None:
+                channel = self._trigger.sender
+            if nick is None:
+                return
```

Comment:
`nick` is a REQUIRED argument, and calling `kick()` without it should fail loudly.

Reply:
I may have missed something elsewhere, but this doesn't appear resolved :confused: 

`return`ing seems more like quitely hiding the problem than "should fail loudly".

Reply:
User would have to explicitly pass in `None`, since the default parameter value was removed. If passed no `nick` argument at all, this will fail quite loudly.

Probably should remove this `if` block entirely.

Reply:
NAK on removing the `if`. Failing silently is fine if explicitly passed `None`, which is in line with how other `bot` API methods work if something required for the command to be valid is None-y.